components/EpisodeListItem/index.js

```diff
@@ -0,0 +1,24 @@
+import Link from "next/link";
+import styled from "styled-components";
+
+const StyledListItem = styled.li`
+  display: flex;
+  flex-direction: row;
+  gap: 2rem;
+  background-color: whitesmoke;
+  margin: 0.5rem;
+  padding: 0.5rem;
+  border: 1px solid;
+  border-radius: 5px;
+`;
+
+export default function EpisodeListItem({ nummer, titel, href }) {
+  return (
+    <Link href={href}>
+      <StyledListItem>
+        <p>#{nummer}</p>
+        <p>...{titel}</p>
+      </StyledListItem>
+    </Link>
+  );
+}
```

Comment:
As your `EpisodeListItem` is rendered inside an unordered list the `Link` component should not be the wrapper of your component. It should be the listitem. Please change that.

Reply:
My intention is to make the whole list item clickable/touchable, not just its children elements. This was the easiest & quickest way for me to achieve this. I am open to suggestions on how to do it in another way.

Reply:
I understand that this was the easiest way, but the html is not correct. [Check caninclude for reference.](https://caninclude.glitch.me/caninclude?child=a&parent=ul)
You can implement the feature like described in [this article](https://adrianroselli.com/2020/02/block-links-cards-clickable-regions-etc.html). Hope this does help! :)

Reply:
Thanks for sharing some references, but unfortunately I didn't really understand the code in the article. I found a solution that works and hope that is correct. (Link styling width: 100%)